src/transformers/api-labels.js

```diff
@@ -0,0 +1,39 @@
+//@ts-check
+
+const visitParents = require("unist-util-visit-parents");
+
+module.exports = function attacher() {
+  return transformer;
+};
+
+const PLATFORMS = ['macOS', 'Windows', 'Linux'];
+const DEPRECATED = 'Deprecated';
+const EXPERIMENTAL = 'Experimental';
+const READONLY = 'Readonly';
+
+/**
+ *
+ * @param {import("unist").Parent} tree
+ */
+async function transformer(tree) {
+  visitParents(tree, 'emphasis', visitor);
+}
+
+  /**
+   *
+   * @param {import("unist").Node} node
+   */
+  function visitor(node) {
+    if (Array.isArray(node.children) && node.children.length === 1) {
```

Comment:
Do we want to filter by node type as well? Will this also transform into a badge a paragraph that only has `Windows`?

I think it would be good for future maintainers to have a small description on where do we expect this transformer to be executed, what it is doing and why.

Maybe something like:

> This transformer should be executed only in the API pages and transforms XYZ into badges to provide visual cues for the platform and status

Reply:
This will only execute when the tree node is:
* an `_emphasis_` node
* with exactly 1 child
* that child's value is specifically one of the strings that we're matching for

e.g. `_macOS_` or `_Windows_`

I think that's precise enough

Reply:
Added documentation in fb2fa3e

Reply:
I completely missed the `_emphasis_` at the top, my bad 😓